mdal/mdal_data_model.hpp

```diff
@@ -38,6 +38,137 @@ namespace MDAL
 
   typedef std::vector< std::pair< std::string, std::string > > Metadata;
 
+  class DateTime;
+
+  class Duration
```

Comment:
I am wondering if we shouldn't use Time and DateTime (similar to QTime and QDataTime), but I am happy with Duration too

Reply:
My point of view:
Time can be confusing and can express a moment or a duration. 
For me QTime express a moment in the day :
"A QTime object contains a clock time, which it can express as the numbers of hours, minutes, seconds, and milliseconds since midnight." (https://doc.qt.io/qt-5/qtime.html#details)
Duration is more explicit and can't express a moment but only a duration.

Reply:
this is naming, i leave it with you